src/cascade/input_data/db/asdr.py

```diff
@@ -95,9 +83,23 @@ def _upload_asdr_data_to_tier_3(execution_context, cursor, model_version_id, asd
         "age_group_years_start": "age_lower",
         "age_group_years_end": "age_upper"
     })
-
     asdr_data = asdr_data.merge(age_group_data, how="left", on="age_group_id")
     asdr_data = asdr_data.where(pd.notnull(asdr_data), None)
```

Comment:
This is an interesting line. It implies some of the asdr data could be null? I should hope not. Not for this PR, I guess, but I wish I knew why this was here. We could warn when this happens.

Reply:
Either Simon or Natalie had a reason, but I can't remember what it was.